learning/inference.py

```diff
@@ -453,6 +484,41 @@ def main():
 
     # BRAINSTORM METRICS FOR **ONLY COMPLETE** RUNS
 
+    chdir(agent.image_directory)
+
+    video_name = PurePath(agent.image_directory).stem
+    filelist_name = video_name + "_filelist.txt"
+
+    image_files = sorted(agent.image_directory.glob("*.png"))
+    with open(filelist_name, "w") as file_out:
+        for file in image_files:
+            file_out.write(f"file '{file}'\nduration 0.0333\n")
+
+    sprun(
+        [
+            "ffmpeg",
+            # "-f" is the format argument and "concat" specifies that the format is to concatenate multiple files
+            "-f",
+            "concat",
+            # "-safe" is the argument of whether to check the input paths for safety and "0" says they don't need to be checked
+            "-safe",
+            "0",
+            # "-i" is the input file argument and "filelist_name" is the file containing the paths to the images that will be concatenated
+            "-i",
+            filelist_name,
+            # "-c:v" sets the codec and specifies it is for video stream and "libx264" specifies the codec to encode the video stream
+            # "-c:v",
+            # "libx264",
+            # # "-pix_fmt" specifies the pixel format for the output video and "yuv420p" is a pixel format using YUV color space and 4:2:0 chroma subsampling
+            # "-pix_fmt",
+            # "yuv420p",
```

Comment:
Don't we need this one?

Reply:
Inference video creation seems to work fine either way with or without it.